src/app/app.component.ts

```diff
@@ -11,9 +10,9 @@ import { LoggingService } from './core/services/logging.service';
   styleUrls: ['./app.component.css']
 })
 export class AppComponent implements AfterViewInit {
-  isNetworkOnline$: Observable<boolean>;
+  NOT_CONNECTED_ERROR: Error = {name: 'Not connected to internet', message: 'You are not connected to the internet.'};
```

Comment:
Let's use a constructor `new Error("message")` for the error here, to keep it consistent across the codebase.